autoload/neomake/makers/ft/nim.vim

```diff
@@ -3,12 +3,13 @@ function! neomake#makers#ft#nim#EnabledMakers()
 endfunction
 
 function! neomake#makers#ft#nim#nim()
-    return {                          
-                \ 'exe': 'nim',                                      
-                \ 'args': ['--verbosity:0', '--colors:off', '-c', 'check'],
-                \ 'errorformat':                                     
-                \   '%I%f(%l\, %c) Hint: %m,' .                      
-                \   '%W%f(%l\, %c) Warning: %m,' .                   
-                \   '%E%f(%l\, %c) Error: %m'                        
-                \ }                                                  
+    return {
+                \ 'exe': 'nim',
```

Comment:
I just kept the previous version, but I prefer to have it explicitely anyway so...